addon/-task-state-mixin.js

```diff
@@ -50,5 +57,18 @@ export default Mixin.create({
 
   _scheduler: null,
 
+  _resetState() {
+    this.setProperties({
+      'last': null,
+      'lastRunning': null,
+      'lastPerformed': null,
+      'lastSuccessful': null,
+      'lastComplete': null,
+      'lastErrored': null,
+      'lastCanceled': null,
+      'lastIncomplete': null,
```

Comment:
Would it make any sense to not reset `lastErrored`, `lastCanceled`, and/or `lastIncomplete`?

Reply:
I think it's best to reset everything back to the state when the task was first initialized; it'd also be easy to forget which properties _didn't_ get reset and why.

Reply:
That makes sense. Just to verify, you are suggesting I leave this as it is now?